Tasks/PackerBuildV0/src/taskInputTemplateVariablesProvider.ts

```diff
@@ -40,6 +40,10 @@ export default class TaskInputTemplateVariablesProvider implements definitions.I
         this._templateVariables.set(constants.TemplateVariableResourceGroupName, taskParameters.resourceGroup);
         this._templateVariables.set(constants.TemplateVariableStorageAccountName, taskParameters.storageAccount);
 
+        if(taskParameters.isManagedImage && taskParameters.templateType === constants.TemplateTypeBuiltin){
```

Comment:
second condition is not necessary. Custom template case is handled above in line no 28.

Reply:
Will do.

Reply:
Done.

Reply:
Done.